pkgs/development/python-modules/pgcli/default.nix

```diff
@@ -44,14 +45,17 @@ buildPythonPackage rec {
     keyring
     pendulum
     sshtunnel
+    setuptools
```

Comment:
Is this caused by switching to pyproject? Then it should go to nativeBuildInputs